src/test/java/javax/validation/ValidationTest.java

```diff
@@ -157,7 +167,7 @@ public CustomValidationProviderClassLoader(String... suffixes) {
 
 		@Override
 		public Enumeration<URL> getResources(String name) throws IOException {
-			CustomEnumeration<URL> customEnumeration = new CustomEnumeration<URL>();
+			CustomEnumeration<URL> customEnumeration = new CustomEnumeration<>();
```

Comment:
Yes, agreed. Even had removed some more of this sort from the non-test code for that reason.